CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Pre-populate Skaffold deployment with substra-tools base docker image (not minimal) ([#736](https://github.com/Substra/substra-backend/pull/736))
 
+### Fixed
+
+- `settings.DEBUG` is not `True` by default.
```

Comment:
```suggestion
- `settings.DEBUG` is now `True` by default.
```

Typo? I'm not sure

Reply:
New default is `False`:
https://github.com/Substra/substra-backend/blob/fix/debug-by-default/backend/backend/settings/common.py#L26

Reply:
OK, I now realize my comment made no sense. What about this

```suggestion
- `settings.DEBUG` is no longer `True` by default.
```

Reply:
RIP suggestion o7